### ISSUE TITLE
When the menu is longer than the question content, the page should be as long as the menu

### ISSUE THREAD

**Issue by christianp** [numbas/Numbas#456]
This applies to the default theme. 
Because the menu is an absolutely positioned element, it doesn't contribute to the page length. So when the menu is longer than the question content, a scroll bar appears in the menu.

**Comment by christianp** [comment]
This is no longer the case.
